pkg/systemd/logs.scss

```diff
@@ -26,6 +20,16 @@
     font-weight: normal;
 }
 
+.journal-entry-value,
+#journal-entry-message {
+    word-break: break-all;
+    white-space: pre-wrap !important;
```

Comment:
I'm assuming the `!important` is needed?

Reply:
yeah, some other rule from patterfly tables is overwriting it. I could use some stronger selector, but that would make it less obvious to what it applies and also there would me more chances how we could break it.